File: src/app/src/main/java/com/onscreensync/tvapp/apiresponses/ConfigApiResponse.java
package com.onscreensync.tvapp.apiresponses;

import com.google.gson.annotations.SerializedName;
import com.onscreensync.tvapp.apiresponses.configs.DisplayApiConfig;

public class ConfigApiResponse {

    @SerializedName("display-api")
    private DisplayApiConfig displayApiConfig;

    public DisplayApiConfig getDisplayApiConfig() {
        return displayApiConfig;
    }
}

File: src/app/src/main/java/com/onscreensync/tvapp/apiresponses/ConfigApiResponse.kt
package com.onscreensync.tvapp.apiresponses

import com.google.gson.annotations.SerializedName
import com.onscreensync.tvapp.apiresponses.configs.DisplayApiConfig

data class ConfigApiResponse(
    @SerializedName("display-api")
    val displayApiConfig: DisplayApiConfig?
)
